File: Jon Agirre/PROGRAMAZIOA/Kapituloa1/Python/Ariketa12/Esfera.py
import math

erradioa = input("Sartu esferaren erradioa: ")

erradioa = float(erradioa)

azalera = (erradioa * erradioa * math.pi * 4)
bolumena = (erradioa * erradioa * erradioa * math.pi * 4) / 3

print("Azalera = ", round(azalera, 6))
print("bolumena = ", round(bolumena, 6))

File: Jon Agirre/PROGRAMAZIOA/Kapituloa1/Java/ariketa12/esfera.java
import java.util.Scanner; 

public class esfera
{
    public static void main(String[] args)
    {
        Scanner sarrera = new Scanner(System.in);

    float erradioa;
    float bolumena;
    float azalera; 

    System.out.print("Sartu esferaren erradioa: ");
    erradioa = sarrera.nextInt();
    
    azalera = (float)(erradioa * erradioa * Math.PI * 4.0);
    
    bolumena = (float)((erradioa * erradioa * erradioa * Math.PI * 4.0) / 3.0);
    
    System.out.printf("Azalera = %.6f\n", azalera);
    
    System.out.printf("Bolumena = %.6f\n",bolumena);
    }
}
